### PR TITLE
fix: enable support for edit_url in TechDocs

### PR DESCRIPTION
**Pull request by cjlee01**
## Hey, I just made a Pull Request!

enable support for edit_url in TechDocs

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by cjlee01** [comment]
enabled edit_url in this pr @awanlin

File: .changeset/tasty-forks-compare.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': patch
+---
+
+Updated getRepoUrlFromLocationAnnotation to check for harness scm integration
```
Comment (awanlin) [inline .changeset/tasty-forks-compare.md:5]:
```suggestion
Updated `getRepoUrlFromLocationAnnotation` to check for Harness SCM integration
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks @cjlee01, just left a suggestions for the changeset that should make Vale happy ;)

File: .changeset/tasty-forks-compare.md (hunk 1 above)
Comment (cjlee01) [reply .changeset/tasty-forks-compare.md]: thanks for review, fixed changeset @awanlin

**Review by freben — APPROVED** [review]

**Review by awanlin — APPROVED** [review]

**Review by johnphilip283 — APPROVED** [review]

**Pull request merged**